README.md

```diff
@@ -9,8 +9,10 @@ Porter gives you building blocks to create a cloud installer for your applicatio
 necessary infrastructure and configuration setup. It is a declarative authoring experience that lets you
 focus on what you know best: your application.
 
+Want to start using Porter? Check out the [Quickstart Guide](docs/content/quickstart.md) for a brief walkthrough.
```

Comment:
The link should go to https://porter.sh/quickstart/

You can preview your doc changes at https://deploy-preview-157--porter.netlify.com/quickstart/ (I got the link from the netlify preview check at the bottom of the PR).